Ported/combat-bees/Assets/Scripts/Authoring/BeeConfigAuthoring.cs

```diff
@@ -12,21 +12,23 @@ class BeeConfigAuthoring : UnityEngine.MonoBehaviour
     public int BeeCount;
     public int FoodCount;
     public GameObject Field;
+    public float3 InitVel;
 }
 
 [UpdateBefore(typeof(NestAuthoring))]
 class BeeConfigBaker : Baker<BeeConfigAuthoring>
 {
     public override void Bake(BeeConfigAuthoring authoring)
     {
+        var renderer = authoring.Field.GetComponent<Renderer>();
         AddComponent<BeeConfig>(new BeeConfig()
         {
             bee = GetEntity(authoring.BeePrefab),
             food = GetEntity(authoring.FoodPrefab),
             beeCount = authoring.BeeCount,
-            foodCount = authoring.FoodCount
+            foodCount = authoring.FoodCount,
+            fieldArea = new Area{Value = AABBExtensions.ToAABB(renderer.bounds)},
```

Comment:
if this is stored in BeeConfig it doesn't need to be an Area component, it should just be a raw AABB